pypesto/petab/importer.py

```diff
@@ -127,76 +127,6 @@ def from_yaml(
             model_name=model_name,
         )
 
-    def check_gradients(
```

Comment:
why is this function completely removed instead of replacing most of it's implementation with a call to `check_gradients_match_finite_differences`

Reply:
See the discussion here: https://github.com/ICB-DCM/pyPESTO/issues/723

Back then I naively introduce the check_gradient(...) method, but in fact this method is superfluous, see the issue #723.